src/anim/controller/anim-blend-tree.js

```diff
@@ -204,6 +94,176 @@ class AnimBlendTree extends AnimNode {
         }
         return count;
     }
+
+    static between(num, a, b, inclusive) {
+        var min = Math.min(a, b),
+            max = Math.max(a, b);
+        return inclusive ? num >= min && num <= max : num > min && num < max;
+    }
+
+    static getAngleRad(a, b) {
+        return Math.atan2(a.x * b.y - a.y * b.x, a.x * b.x + a.y * b.y);
+    }
+
+    static clamp(num, min, max) {
+        return num <= min ? min : num >= max ? max : num;
+    }
+
+}
+
+/**
+ * @private
+ * @class
+ * @name AnimBlendTree1D
+ * @classdesc An AnimBlendTree that calculates it's weights using a 1D algorithm
+ * @description Create a new BlendTree1D.
+ */
+class AnimBlendTree1D extends AnimBlendTree {
```

Comment:
move this into separate file please

Reply:
Sure, i've moved them all to separate files now.